scripts/gen-function/parse.ts

```diff
@@ -49,11 +52,11 @@ export function parse(content: string): Definition[] {
  */
 function parseBlock(fn: string, body: string): Definition {
   // Remove '\n' in {variant} to make {variant} single line (ex. `searchpairpos`)
-  body = body.replaceAll(new RegExp(`^\(${fn}\\([^\)]*?\)\n\t*`, "g"), "$1");
+  body = body.replaceAll(new RegExp(`^(${fn}\\([^)]*?)\\n\\t*`, "gm"), "$1");
   // Append ')' for an invalid {variant}. (ex. `win_id2tabwin` in Neovim)
-  body = body.replaceAll(new RegExp(`^\(${fn}\\([^\)]*?\)\t+`, "g"), "$1)\t");
+  body = body.replaceAll(new RegExp(`^(${fn}\\([^)]*?)\\t+`, "gm"), "$1)\t");
   // Insert '\n' between {variant} and {document} (ex. `argidx`)
-  body = body.replaceAll(new RegExp(`^\(${fn}\\(.*?\\)\)\t`, "g"), "$1\n\t\t");
+  body = body.replaceAll(new RegExp(`^(${fn}\\(.*?\\))\\t`, "gm"), "$1\n\t\t");
```

Comment:
It is important to use multiline mode with `m`.
Regular expression escaping was fixed.